src/test/java/org/generationcp/ibpworkbench/actions/RunMultiSiteActionTest.java

```diff
@@ -396,4 +392,9 @@ private DataSet createSummaryDataSet() {
 
 	}
 
+	private boolean isValidFileNameFormat(final String fileName, final String pattern) {
```

Comment:
How about extracting `isValidFileNameFormat` as public method in `FileNameGnerator` class?